dipy/sims/tests/test_phantom.py

```diff
@@ -55,32 +55,32 @@ def test_phantom():
     FA[np.isnan(FA)] = 0
     # 686 -> expected FA given diffusivities of [1500, 400, 400]
     l1, l2, l3 = 1500e-6, 400e-6, 400e-6
-    expected_fa =  (np.sqrt(0.5) * np.sqrt((l1 - l2)**2 + (l2-l3)**2 + (l3-l1)**2 )/np.sqrt(l1**2 + l2**2 + l3**2))
+    expected_fa = np.sqrt(0.5) * np.sqrt((l1 - l2)**2 + (l2 - l3)**2 + (l3 - l1)**2) / np.sqrt(l1**2 + l2**2 + l3**2))
 
-    assert_array_almost_equal(FA.max(), expected_fa, decimal=2)
+    assert_array_almost_equal(FA.max(), expected_fa, decimal = 2)
 
 
 def test_add_noise():
     np.random.seed(1980)
 
-    N = 50
-    S0 = 100
+    N=50
+    S0=100
 
-    options = dict(func=f,
-                   t=np.linspace(0, 2 * np.pi, N),
-                   datashape=(10, 10, 10, len(bvals)),
-                   origin=(5, 5, 5),
-                   scale=(3, 3, 3),
-                   angles=np.linspace(0, 2 * np.pi, 16),
-                   radii=np.linspace(0.2, 2, 6),
-                   S0=S0)
+    options=dict(func = f,
+                   t = np.linspace(0, 2 * np.pi, N),
+                   datashape = (10, 10, 10, len(bvals)),
+                   origin = (5, 5, 5),
+                   scale = (3, 3, 3),
+                   angles = np.linspace(0, 2 * np.pi, 16),
+                   radii = np.linspace(0.2, 2, 6),
+                   S0 = S0)
```

Comment:
No spaces around "=" sign when assigning to a dict or when passing arguments to a function, please.